cherrypicks_test.go

```diff
@@ -121,9 +121,9 @@ Hello :smile_cat: This PR contains changelog entries. Please, verify the need of
 			comment: &github.IssueComment{
 				Body: github.String(`
 Hello :smile_cat: This PR contains changelog entries. Please, verify the need of backporting it to the following release branches:
+2.1.x (release 3.4.x)
 2.0.x (release 3.3.x)
 2.0.x (release 3.2.x)
```

Comment:
I guess, technically `3.2` is no longer supported

Reply:
I checked this with @snusifer and the issue is that the unit tests do not fetch `releases/versions.js` and instead fallback to the old method of using `release_tool`. That is why there are present here.